tests/analysis/Decompilation/dynamic_native/array.c

```diff
@@ -0,0 +1,27 @@
+#include <stdint.h>
+
+struct __attribute__((packed)) X {
+  uint16_t a[10];
+  uint64_t b;
+};
```

Comment:
Can you come up with some more meaningful naming? Also let's be coherent in coding style.
In C, usually we go like python. Snake case, first letter lowercase.

Something like this?
```c
typedef struct {
  uint16_t uint16_array[10];
  uint64_t uint64;
} X;
```

Reply:
Only for variable and fields or also for `typedef`s?

Reply:
I tried to improve naming, please check if it's ok.